test/suite/test_rollback_to_stable01.py

```diff
@@ -26,18 +26,41 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import wiredtiger, wttest
+import os, subprocess, wiredtiger, wttest
 from wtdataset import SimpleDataSet
 from wiredtiger import stat, wiredtiger_strerror, WiredTigerError, WT_ROLLBACK
 from wtscenario import make_scenarios
 from time import sleep
 
+def get_git_root():
+    output = subprocess.run(["git", "rev-parse", "--show-toplevel"], check=True, capture_output=True)
+    return output.stdout.strip().decode("utf-8")
+
+def verify_rts_logs():
```

Comment:
Could you move this to a different file? Maybe wt_rts.py?

Reply:
Good call, thanks. Done in 9c23605.